react-components/src/architecture/concrete/observationsDomainObject/ObservationsTool.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type IconType } from '@cognite/cogs.js';
+import { type TranslateKey } from '../../base/utilities/TranslateKey';
+import { type FdmSDK, type InstanceFilter } from '../../../utilities/FdmSDK';
+import { OBSERVATION_SOURCE, type Observation, type ObservationProperties } from './models';
+import { ObservationsDomainObject } from './ObservationsDomainObject';
+import { Color, Vector3 } from 'three';
+import { CDF_TO_VIEWER_TRANSFORMATION, type Overlay3D, Overlay3DCollection } from '@cognite/reveal';
+import { NavigationTool } from '../../base/concreteCommands/NavigationTool';
+import { type RevealRenderTarget } from '../../base/renderTarget/RevealRenderTarget';
+
+const DEFAULT_OVERLAY_COLOR = new Color('lightblue');
+const SELECTED_OVERLAY_COLOR = new Color('red');
```

Comment:
just a cosmetic question: is it not better to move out these const to a `constant` file?

Reply:
Sure, I don't have a strong opinion :sweat_smile: